### ISSUE TITLE
cannot get back to exact location of bookmark with label using NVDA

### ISSUE THREAD

**Issue by GeorgeKerscher** [edrlab/thorium-reader#3827]
I am using Thorium 3.5.1 beta 1 on Windows 11 with NVDA 2026.2.
I have native mode turned on, and I  hit nvda key plus enter, which should sync the virtual buffer with the display. Also highlight a linne and copy to the clipboard. When I leave a bookmark with a label, I can enter text, select a color and save.

However when I navigate back to the bookmark with label, I am not taken to the location of the bookmark. I am in the current file.

I can leave another bookmark in a different file in the EPUB, and I am taken to that file, but not to the spot where I left the bookmark.

I mark this as a pass for bookmarks, because I could paste the text I copied and use the NVDA search feature to get back to the exact location, but this is a work around.
@prashantverma2014
@ways2read

**Comment by danielweck** [comment]
Thank you George. I wonder why the bookmark hyperlink destination (when activated from the navigation dialog, I assume) is not exactly the HTML element containing the bookmark. It should be no different from following a hyperlink / heading label from the table of contents, or a "goto page break" or a landmark item from the authored list of landmarks. I will test again with VoiceOver but I remember this working fine.

**Comment by GeorgeKerscher** [comment]
I just tested again. I have two bookmarks with labels. I select either one, the button to go to the content, and focus does not move at all. I stay on that button. I tried shift + F10 and ctrl + shift + f10 and that does nothing.

When I navigate back to the document I am left where I was before I evocked the shortcut to show the list of bookmarks.

**Comment by GeorgeKerscher** [comment]
I just downloaded 3.53.1 and the bookmark function is working. I have to press ctrl + shift + F10 to go there directly.

Closing this issue.
